prompt_engineering/config.yaml

```diff
@@ -1,4 +1,8 @@
-api: "sambaverse"
+api: "sambastudio" #  set either sambastudio, sambaverse or fastapi
+
+llm:
+  max_tokens_to_generate: 500
+  temperature: 0.0
```

Comment:
can we add the above api as llm sub key?

Reply:
done! didn't know this was the new standard, but I've added a couple of PRs that support this change. Thanks!